pkg/runtime/cli.go

```diff
@@ -30,19 +30,24 @@ func FromFlags() (*DaprRuntime, error) {
 	daprInternalGRPCPort := flag.String("dapr-internal-grpc-port", "", "gRPC port for the Dapr Internal API to listen on")
 	appPort := flag.String("app-port", "", "The port the application is listening on")
 	profilePort := flag.String("profile-port", fmt.Sprintf("%v", DefaultProfilePort), "The port for the profile server")
-	appProtocol := flag.String("protocol", string(HTTPProtocol), "Protocol for the application: grpc or http")
+	appProtocol := flag.String("app-protocol", "", "Protocol for the application: grpc or http")
```

Comment:
This is a breaking change. Why did we remove the default being HTTP?

Reply:
This isn't a breaking change. The default here is removed as we need logic to be backward compatible. Please look at [this section](https://github.com/dapr/dapr/pull/1865/files#diff-7fadd8b154bfb3e50073e295c476aa6eR283-R289) where I made HTTP to be default.

Reply:
Defaults should come from `cli.go` so that they're visible and explicit.

Reply:
Makes sense, changed it.

Reply:
@yaron2 good to merge?